.github/workflows/build_and_test.yml

```diff
@@ -232,8 +232,15 @@ jobs:
       run: |
         cd Dist && ALL_LIBRARIES_WITHOUT_SYMLINKS=($(find . -name "*.a" -type f)) &&  for LIBRARY in ${ALL_LIBRARIES_WITHOUT_SYMLINKS[@]}; do echo "Checking for arm64 in $LIBRARY"; CHECK_ARCH=$(lipo $LIBRARY -verify_arch arm64 2>&1); RETVAL=$?; if [ $RETVAL -eq 1 ]; then echo "Couldn't find arm64 slice in $LIBRARY"; exit 1; else echo "Found arm64 slice in $LIBRARY"; fi; done
 
-  mingw:
+  mingw-w64:
     runs-on: windows-latest
+    strategy:
+      matrix:
+        include:
+         - env: i686
+           sys: mingw32
+         - env: x86_64
+           sys: mingw64
```

Comment:
I'm not sure this is a valid configuration because WinPcap only supports 32-bit in MinGW (as mentioned [here](https://pcapplusplus.github.io/docs/install/build-source/mingw#additional-prerequisites))

Reply:
In this build we are linking with NPcap not WinPcap.

Winpcap seems no more maintained regarding this 
https://npcap.com/vs-winpcap.html

Reply:
oh, you're right!